app/src/main/java/com/duckduckgo/app/statistics/api/OfflinePixelSender.kt

```diff
@@ -33,15 +37,17 @@ import javax.inject.Inject
  */
 class OfflinePixelSender @Inject constructor(
     private val dataStore: OfflinePixelDataStore,
```

Comment:
With the new bahavior we need to be more specific with the `dataStore` name and class name. Perhaps it could become `offlineCountPixelDataStore: OfflineCountPixelDataStore`